code/workspaces/web-app/src/containers/adminResources/AdminResourcesContainer.js

```diff
@@ -12,16 +12,22 @@ import ProjectResources from './ProjectResources';
 import PromisedContentWrapper from '../../components/common/PromisedContentWrapper';
 import Pagination from '../../components/stacks/Pagination';
 
-const useStyles = makeStyles(theme => ({
-  showControls: {
+const useShowControlsStyles = makeStyles(() => ({
```

Comment:
Just wondered why this has been refactored like this to separate out the component styling?

Reply:
It was picked up by the TypeScript static analysis.  `makeStyle` returns a style for a component (as root), with additional styles for nested components - see https://material-ui.com/styles/basics/#hook-api.  Nothing was acting as the 'root' style, so I separated them out.